moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -413,8 +421,16 @@ void JogCalcs::insertRedundantPointsIntoTrajectory(trajectory_msgs::JointTraject
   }
 }
 
-void JogCalcs::lowPassFilterPositions(sensor_msgs::JointState& joint_state)
+void JogCalcs::lowPassFilterPositions(sensor_msgs::JointState& joint_state, JogArmShared& shared_variables)
 {
+  // Check if a new filter coefficient was supplied
+  if (!shared_variables.filter_coefficient_queue.empty())
```

Comment:
Not sure whether it could be an issue here but calling `empty` on a lockfree queue isn't accurate in general. Boost documentation has the following note: 

> The result is only accurate, if no other thread modifies the queue. Therefore it is rarely practical to use this value in program logic.